cmd/imagec/imagec.go

```diff
@@ -379,6 +382,17 @@ func WriteImageBlobs(images []*ImageWithMeta) error {
 	if err := os.RemoveAll(destination); err != nil {
 		return fmt.Errorf("Failed to remove download directory: %s", err)
 	}
+
+	if len(images) > 0 && options.imagefile != "" {
```

Comment:
This needs to be fixed to cover the case of pulling the same image by a different tag or by the digest. In this current implementation, the image metadata would not get updated with the new tag information. I will consult with @cgtexmex and see if I can come up with something to address this.